charts/__tests__/gardener-dashboard/runtime/dashboard/__snapshots__/deployment.spec.js.snap

```diff
@@ -146,6 +146,10 @@ Object {
                 "name": "GARDENER_CONFIG",
                 "value": "/etc/gardener-dashboard/config.yaml",
               },
+              Object {
```

Comment:
I assume the chart tests have failed or why this is part of your PR?

Reply:
Yes they fail with current master